components/service/glean/build.gradle

```diff
@@ -19,6 +31,11 @@ android {
         release {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            resValue("string", "GLEAN_PING_SCHEMA_URL", GLEAN_PING_SCHEMA_URL)
+        }
+
+        debug {
+            resValue("string", "GLEAN_PING_SCHEMA_URL", GLEAN_PING_SCHEMA_URL)
```

Comment:
Maybe instead of a String resource you want to use just a `BuildConfig` constant. If you define it in the `defaultConfig` block then you won't need to duplicate it (See `LIBRARY_VERSION` example above).